lib/commit.rb

```diff
@@ -41,10 +41,7 @@ def stats
       df.set_repository(@jrepo)
       df.set_context(0)
       parent_commit = @jcommit.parent_count > 0 ? @jcommit.get_parents[0] : nil
-
-      rename_detector = RenameDetector.new(@jrepo)
-      rename_detector.addAll(df.scan(parent_commit, @jcommit))
-      entries = rename_detector.compute
+      entries = df.scan(parent_commit, @jcommit)
```

Comment:
Shouldn't we then set `setDetectRenames(true)`?

Reply:
I tested this and it's already true! The rename spec passes so all is fine :)

Reply:
OK.

Reply:
Strangely I didn't need to explicitly enable `setDetectRenames` locally, but the test on Travis was failing. So added it anyway.